diff --git a/node_modules/photoswipe/.DS_Store b/node_modules/photoswipe/.DS_Store
new file mode 100644
index 0000000..6925c29
Binary files /dev/null and b/node_modules/photoswipe/.DS_Store differ
diff --git a/node_modules/photoswipe/dist/photoswipe-ui-default.js b/node_modules/photoswipe/dist/photoswipe-ui-default.js
index f3733e1..03c7cf5 100644
--- a/node_modules/photoswipe/dist/photoswipe-ui-default.js
+++ b/node_modules/photoswipe/dist/photoswipe-ui-default.js
@@ -47,7 +47,7 @@ var PhotoSwipeUI_Default =
 
 		_options,
 		_defaultUIOptions = {
-			barsSize: {top:44, bottom:'auto'},
+			barsSize: {top:75, bottom:'auto'},
 			closeElClasses: ['item', 'caption', 'zoom-wrap', 'ui', 'top-bar'], 
 			timeToIdle: 4000, 
 			timeToIdleOutside: 1000,
diff --git a/node_modules/photoswipe/dist/photoswipe.js b/node_modules/photoswipe/dist/photoswipe.js
index 467c1ce..747c99f 100644
--- a/node_modules/photoswipe/dist/photoswipe.js
+++ b/node_modules/photoswipe/dist/photoswipe.js
@@ -2721,11 +2721,12 @@ var _getItemAt,
 			min:{x:0,y:0}
 		};
 	},
-	_calculateSingleItemPanBounds = function(item, realPanElementW, realPanElementH ) {
+	_calculateSingleItemPanBounds = function(item, realPanElementW, realPanElementH, viewportSize) {
 		var bounds = item.bounds;
-
+		let extraMargin = 0;
+		if(viewportSize.x > 1024) extraMargin = 40;
 		// position of element when it's centered
-		bounds.center.x = Math.round((_tempPanAreaSize.x - realPanElementW) / 2);
+		bounds.center.x = Math.round((_tempPanAreaSize.x - realPanElementW) / 2) + extraMargin;
 		bounds.center.y = Math.round((_tempPanAreaSize.y - realPanElementH) / 2) + item.vGap.top;
 
 		// maximum pan position
@@ -2738,7 +2739,7 @@ var _getItemAt,
 							bounds.center.y;
 		
 		// minimum pan position
-		bounds.min.x = (realPanElementW > _tempPanAreaSize.x) ? 0 : bounds.center.x;
+		bounds.min.x = (realPanElementW > _tempPanAreaSize.x) ? 0 : bounds.center.x + extraMargin;
 		bounds.min.y = (realPanElementH > _tempPanAreaSize.y) ? item.vGap.top : bounds.center.y;
 	},
 	_calculateItemSize = function(item, viewportSize, zoomLevel) {
@@ -2756,13 +2757,14 @@ var _getItemAt,
 
 
 			_tempPanAreaSize.x = viewportSize.x;
-			_tempPanAreaSize.y = viewportSize.y - item.vGap.top - item.vGap.bottom;
+			_tempPanAreaSize.y = viewportSize.y - item.vGap.top * 2;
 
 			if (isInitial) {
-				var hRatio = _tempPanAreaSize.x / item.w;
+				var hRatio = (_tempPanAreaSize.x - 288 - 40) / item.w;
 				var vRatio = _tempPanAreaSize.y / item.h;
 
 				item.fitRatio = hRatio < vRatio ? hRatio : vRatio;
+debugger;
 				//item.fillRatio = hRatio > vRatio ? hRatio : vRatio;
 
 				var scaleMode = _options.scaleMode;
@@ -2789,7 +2791,7 @@ var _getItemAt,
 				return;
 			}
 
-			_calculateSingleItemPanBounds(item, item.w * zoomLevel, item.h * zoomLevel);
+			_calculateSingleItemPanBounds(item, item.w * zoomLevel, item.h * zoomLevel, viewportSize);
 
 			if (isInitial && zoomLevel === item.initialZoomLevel) {
 				item.initialPosition = item.bounds.center;
